### PR TITLE
Relations in Hibernate

### PR DESCRIPTION
**Pull request by renukaThink**
@OneToOne,@OneToMany,@ManyToOne,@ManyToMany annotations with implementation,Lazy,Eager,Second Level Cache

### CODE REVIEW

File: day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.example.mappingtypes.manytomany;
+
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Cacheable
+@org.hibernate.annotations.Cache(usage= CacheConcurrencyStrategy.READ_ONLY)
```
Comment (yogesh-think) [inline day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java:10]: Can we use @Cache instead @org.hibernate.annotations.Cache?

File: day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @Id
+    private  int eid;
+    private String ename;
+    @ManyToMany//(fetch = FetchType.EAGER)
+//    @JoinTable(name = "ProjectsToEmployee",
+//            joinColumns = { @JoinColumn(name="eid")},
+//            inverseJoinColumns = {@JoinColumn(name = "pid")})
+    private List<Projects> projects;
+
+    public Employees() {
```
Comment (yogesh-think) [inline day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java:21]: No need to write non parameterized constructor

File: day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java (hunk 1 above)
Comment (renukaThink) [reply day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java:10]: yes we can use

File: day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java (hunk 2 above)
Comment (renukaThink) [reply day-4/src/main/java/org/example/mappingtypes/manytomany/Employees.java:21]: okay

**Pull request merged**